docs/Parameters.md

```diff
@@ -54,8 +54,8 @@ The parameter format is `key1=value1 key2=value2 ... ` . And parameters can be s
   * support multi validation data, separate by `,`
 * `num_iterations`, default=`100`, type=int, alias=`num_iteration`,`num_tree`,`num_trees`,`num_round`,`num_rounds`
   * number of boosting iterations
-  * note: `num_tree` here equal with `num_iterations`. For multi-class, it actually learns `num_class * num_iterations` trees.
-  * note: For python/R package, cannot use this parameters to control number of iterations.
+  * note: For python/R package, **this parameters is ignored**.
```

Comment:
@Laurae2 Do you mean that `note: For python/R package, **this parameters is ignored**.` is equal to `note: For python/R package, cannot use this parameters to control number of iterations.` ?
I do agree it is equal. My purpose was to rephrase it (I think my version uses more common language) and make it more visible (now it's a 2nd point in a list, not 3rd, and now it's bold).